CONTRIBUTING.md

```diff
@@ -107,14 +107,13 @@ with what you'd like to do or have ideas that will help you do it.
 ### Adding a new library
 
 If you want to submit type stubs for a new library, you need to
-**contact the maintainers of the original library** first to let them
-know and **get their permission**.  Do it by opening an issue on their
+**contact the maintainers of the original library** to let them
```

Comment:
I have been thinking whether it would be better to have us (the typeshed maintainers) do the notification instead, so we can make it even easier for contributors to contribute stubs without worrying about formalities. What do you think?

Reply:
Good idea. We could use a standard template for that, which could contain more information than another contributor would normally provide. (Links to projects, the "type school" if that happens etc.)

Reply:
If people feel it's important to do the notification then having typeshed do it would be the lowest barrier to entry, but it won't scale if this catches on, so what I would do is remove the requirement and just make it a best practice that is okay to drop if it turns out to be too much of a burden.